CONTRIBUTING.md

```diff
@@ -30,9 +30,9 @@ After our team has investigated each issue, we will label them as:
 
 ## Contribute Code
 
-We welcome pull requests aimed at fixing bugs and security issues.
+We welcome Pull Requests aimed at fixing bugs and security issues. Refactoring, product changes and other features won't be considered and therefore the Pull Request will be closed.
```

Comment:
Posted a comment about this bit https://app.asana.com/0/0/1204558888185157/1204888958021886/f